ui/react-components/components/AppointmentEditor/AppointmentEditor.jsx

```diff
@@ -112,7 +120,7 @@ const AppointmentEditor = props => {
         return recurringPattern;
     };
 
-    const getAppointment = () => {
+    const getAppointmentRequest = () => {
         let appointment = {
             patientUuid: appointmentDetails.patient && appointmentDetails.patient.uuid,
```

Comment:
Can we have a model class called 'appointment' and move logic specific to that object to that class?

Reply:
We will handle this as a separate tech debt card.